grafana/dashboard.json

```diff
@@ -0,0 +1,646 @@
+{
+    "__inputs": [
+        {
+            "name": "DS_PROMETHEUS",
+            "label": "Prometheus",
+            "description": "",
+            "type": "datasource",
+            "pluginId": "prometheus",
+            "pluginName": "Prometheus"
+        }
+    ],
+    "__requires": [
+        {
+            "type": "grafana",
+            "id": "grafana",
+            "name": "Grafana",
+            "version": "6.2.1"
```

Comment:
can we test it for `6.0.0`? otherwise, we mandate the users to have the most recent version of Grafana. I'm not sure that most of the user keep their grafana up-to-date.

Reply:
Yeah we can do testing with 5 too, I think we are not using anything from version 6 , I'll double check

Reply:
Works with 5.0.0 👍